pkg/scheduler/internal/cache/node_tree.go

```diff
@@ -74,6 +75,7 @@ func (nt *NodeTree) AddNode(n *v1.Node) {
 	nt.mu.Lock()
 	defer nt.mu.Unlock()
 	nt.addNode(n)
+	nt.recomputeAllNodes()
```

Comment:
nit: I would put `recomputeAllNodes` in `addNode` and `removeNode`, instead of `AddNode` and `RemoveNode`.

Reply:
done